src/Controls/Geosearch.js

```diff
@@ -216,8 +219,15 @@ export var Geosearch = Control.extend({
       this._input.placeholder = this.options.placeholder;
     }
 
+    // create the main suggested results container element
     this._suggestions = DomUtil.create('div', 'geocoder-control-suggestions leaflet-bar', this._wrapper);
 
+    // create a child contents container element for each provider inside of this._suggestions
+    // to maintain the configured order of providers for suggested results
+    for (var i = 0; i < this.options.providers.length; i++) {
+      this.options.providers[i]._contentsElement = DomUtil.create('span', null, this._suggestions);
```

Comment:
this is just a nit, but semantically I'd expect the container for each provider to be a `<div>` too.

I see above that we've been using a `<span>` for individual headers too though, so no need to go rearranging deck chairs, especially if it makes more .css work.

Reply:
I agree and will give `<div>`s the chance they deserve both with these new elements and the existing ones.

Reply:
Actually worked like a charm without any css changes...yet.  The new structure per each provider is:

```html
<div>
  <div class="geocoder-control-header"></div>
  <ul class="geocoder-control-list"></ul>
<div>
```

Reply:
now that's what I call _semantic_.